web/client/plugins/MetadataExplorer.jsx

```diff
@@ -259,6 +261,17 @@ class MetadataExplorerComponent extends React.Component {
     }
 }
 
+const MetadataExplorerComponentWrapper = (props, context) => {
+    const { loadedPlugins } = context;
+    const addonsItems = usePluginItems({ items: props.items, loadedPlugins }).filter(({ target }) => target === 'url-addon');
+    return <MetadataExplorerComponent {...props} addonsItems={addonsItems}/>;
+};
+
+
+MetadataExplorerComponentWrapper.contextTypes = {
```

Comment:
If we are going in the direction of injected plugin we need documentation on how to use the `url-addon` items

Reply:
will